speech/api/Recognize/Recognize.cs

```diff
@@ -674,6 +700,7 @@ public static int Main(string[] args)
                 >(args).MapResult(
                 (SyncOptions opts) => IsStorageUri(opts.FilePath) ?
                     SyncRecognizeGcs(opts.FilePath) : opts.EnableWordTimeOffsets ?
+                    SyncRecognizeWithProfanityFilterGcs(opts.FilePath) : opts.EnableProfanityFilter ?
                     SyncRecognizeWords(opts.FilePath) : opts.EnableAutomaticPunctuation ?
```

Comment:
The way you have it, `SyncRecognizeWithProfanityFilterGcs` will execute if `EnableWordTimeOffsets` and `SyncRecognizeWords` will execute if `EnableProfanityFilter`, I belive you want the other way around, like this:

```suggestion
                    SyncRecognizeGcs(opts.FilePath) : opts.EnableWordTimeOffsets ?
                    SyncRecognizeWords(opts.FilePath) : opts.EnableProfanityFilter ?
                    SyncRecognizeWithProfanityFilterGcs(opts.FilePath) : opts.EnableAutomaticPunctuation ?
```